sunxi-tools: (new)

elitak opened the pull request:
This is my first submission to nixpkgs, so please afford these changes some extra scrutiny.

I have tested each build manually.

Some concerns:
- Do I need to make myself the maintainer somehow?
- Am I submitting this to the correct branch?
- Have I versioned default.nix and 1.3.nix correctly here? (they are both sunxi-tools-1.3, but the former uses a newer git revision).

elitak committed:
sunxi-tools: init at 1.3

jagajaga commented on pkgs/development/tools/sunxi-tools/1.2.nix:
```diff
@@ -0,0 +1,28 @@
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Tools for Allwinner A10 devices.";
```
Please remove dot at the end of the description.

jagajaga commented on pkgs/development/tools/sunxi-tools/1.3.nix:
```diff
@@ -0,0 +1,28 @@
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Tools for Allwinner A10 devices.";
```
Here too.

jagajaga commented on pkgs/development/tools/sunxi-tools/default.nix:
```diff
@@ -0,0 +1,28 @@
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Tools for Allwinner A10 devices.";
```
And here.

jagajaga commented:
Is there a reason we need so many versions of this pkg?

jagajaga commented:
Yes, if you want to, you can become a maintainer.

elitak commented:
There's probably no need for older versions, so I've removed them. I wasn't aware fewer versions was preferred and was including them for completeness.

I've added myself as the maintainer.

jagajaga commented:
Please squash commits into one.

elitak commented:
Okay, done.

elitak commented:
I also took the liberty of fixing an out-of-lexographical-order alias in maintainers.nix.

jagajaga commented:
Please rename commit to `sunxi-tools: init at 1.3`

elitak commented:
Done.
